packages/storage-plus/src/keys.rs

```diff
@@ -74,6 +75,33 @@ impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a>> PrimaryKey<'a> for
     }
 }
 
+// use generics for combining there - so we can use &[u8], PkOwned, or IntKey
+impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a> + Prefixer<'a>, V: PrimaryKey<'a>>
+    PrimaryKey<'a> for (T, U, V)
+{
+    type Prefix = T;
+
+    fn key(&self) -> Vec<&[u8]> {
+        let mut keys = self.0.key();
```

Comment:
Huh, how does this work in the end? Isn't `self.0.key()` a `Vec<u8>`?

Reply:
Yes. This is just a generalization of the `(T, U)` case above. Same code.